src/game.cpp

```diff
@@ -7278,6 +7281,25 @@ void game::zones_manager()
     refresh_all();
 }
 
+void game::pre_print_all_tile_info( const tripoint &lp, const catacurses::window &w_info,
+                                    int column, int &first_line, const int last_line,
```

Comment:
So what about `column` and `draw_terrain_indicators` - they aren't used other than in redundant checks. Would you remove them?